front/app/components/UI/Modal/index.tsx

```diff
@@ -468,7 +455,8 @@ class Modal extends PureComponent<Props, State> {
       .subscribe((event) => {
         if (event.target) {
           const height = event.target['innerHeight'] as number;
-          this.setState({ windowHeight: height });
+          const width = event.target['innerWidth'] as number;
+          this.setState({ windowWidth: width, windowHeight: height });
```

Comment:
This is the same as what GetWindowSize did.

Reply:
Does it work on resize?

Reply:
Yes!